testutil/wait_test.go

```diff
@@ -12,32 +12,37 @@ import (
 
 func TestWait_WaitForFilesUntil(t *testing.T) {
 
-	var files []string
-	for i := 1; i < 10; i++ {
-		filename := fmt.Sprintf("test%d.txt", i)
-		filepath := filepath.Join(os.TempDir(), filename)
```

Comment:
Conflicts with some other test writing directly in `/tmp`; at the very least we can make this test use a custom dir